asteroid/filterbanks/melgram_fb.py

```diff
@@ -0,0 +1,61 @@
+import torch
+from . import STFTFB
+from . import transforms
+
+
+class MelGramFB(STFTFB):
+    """Mel magnitude spectrogram filterbank.
+
+    Args:
+        n_filters (int): Number of filters. Determines the length of the STFT
+            filters before windowing.
+        kernel_size (int): Length of the filters (i.e the window).
+        stride (int, optional): Stride of the convolution (hop size). If None
+            (default), set to ``kernel_size // 2``.
+        window (:class:`numpy.ndarray`, optional): If None, defaults to
+            ``np.sqrt(np.hanning())``.
+        sample_rate (float): Sample rate of the expected audio.
+            Defaults to 8000.
+        n_mels (int): Number of mel bands.
+        fmin (float): Minimum frequency of the mel filters.
+        fmax (float): Maximum frequency of the mel filters.
+        **kwargs:
+    """
+
+    def __init__(
```

Comment:
Are you sure about the default of 128 for `n_mels`?
With a sample rate of 8kHz, we could imagine a use case of having `kernel_size` of 20 * 8 = 160, which would result in frequency bins that are 25Hz apart. With 128 mel bands, we would have the first few centres at ~0Hz, 10.5Hz and 21Hz (not sure about the values used by librosa as my current internet connection won't allow me to download the package). In other words, there may be some mel filterbanks that span the same frequency bins.

Reply:
Good question. It's a bit tricky as it depends on the frequency resolution, which depends on `kernel_size`. In this context, the mel magnitude spectrum will be used for separation, which, I guess, may be easier it there are more frequency bands? So I guess 64 should do (and be ok for short frames as well).

Reply:
40 mels is pretty much the standard value for fbanks for ASR and diarization (x-vectors)